pyproject.toml

```diff
@@ -8,7 +8,7 @@ requires = [
     "wheel",
     "Cython~=0.29.32",
     "numpy~=1.21.0",
-    "torch@https://download.pytorch.org/whl/lts/1.8/cu111/torch-1.8.2%2Bcu111-cp38-cp38-linux_x86_64.whl"
+    "torch@https://download.pytorch.org/whl/cu116/torch-1.13.1%2Bcu116-cp38-cp38-linux_x86_64.whl"
```

Comment:
@JihwanEom I'd like to remove this item. This is only required for `mpl` torch extension build. Could you create a draft PR to replace the custom operation by default one (I don't remember what it is)?

Reply:
Created #1620, please refer this PR.